extensions/gitpod-web/webview-ui/public/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>Hello World</title>
+	<meta charset='utf-8'>
+	<meta name='viewport' content='width=device-width,initial-scale=1'>
+	<link rel='stylesheet' href='/build/bundle.css'>
+	<script defer src='/build/bundle.js'></script>
+</head>
+<body>
+</body>
+</html>
```

Comment:
Delete this?

Reply:
We can debug with webview-ui only (not run install built-in extensions) with this `index.html` file

Reply:
mmm is it really useful? I ask that because vscode will inject the API and css variables, so if you run it with this html those won't be defined

Reply:
😂 Yes, I'm not using this too, I create a extension locally to debug webview